WcaOnRails/spec/features/competition_management_spec.rb

```diff
@@ -118,15 +118,15 @@
     end
 
     scenario "select anyone free guest entry status" do
-      competition = FactoryBot.create(:competition, :with_delegate, id: "OldId2016", free_guest_entry_status: 1)
+      competition = FactoryBot.create(:competition, :with_delegate, id: "OldId2016", guest_entry_status: 1)
```

Comment:
Access the enum `Competition.guest_entry_statuses['restricted']` like the way you do in the factory method?

Reply:
Was trying to minimize the changeset for `free_guest_entry_status` -> `guest_entry_status`. Will do in a separate commit!